gangplank/ocp/filer.go

```diff
@@ -126,11 +127,21 @@ func newMinioServer(cfgFile string) *minioServer {
 
 // GetClient returns a Minio Client
 func (m *minioServer) client() (*minio.Client, error) {
+	region := m.Region
+	var secure bool
+	if m.ExternalServer {
+		if strings.Contains(m.Host, "aws") {
```

Comment:
should we match more? 

```suggestion
		if strings.Contains(m.Host, "s3.amazonaws.com") {
```

Reply:
Dropped the list buckets and added better matching.

FWIW, the check on the bucket happens early enough and with the reading of the meta-data over the Minio channel the check is largely unneeded.